docs/release.md

```diff
@@ -35,3 +35,42 @@ refer to `X.Y.Z` as _major_, _minor_ and _patch_ version.
    changes.
 
 1. Click on the green button `Publish release`
+
+## Testing the release build pipeline
```

Comment:
Is this already part of CI? Or is this something we're expected to perform manually before publishing?

Reply:
Great question!

This is not part of CI, we're expected to perform this manually when making changes to the release build pipeline (the GitHub Actions workflow).

I.e. this is a manual check to verify that the release build pipeline works as expected.

Do you think it's worth automating this check and running it for every release?

Reply:
I think it's fine to keep it manual, but can you please document that this is to be performed when working on the pipeline? That wasn't clear to me

Reply:
Ie "testing the release build pipeline" could also mean it's something you're supposed to do after every release, to verify the release build pipeline worked